README.md

```diff
@@ -41,6 +41,16 @@ Other than that, as `sbt-dynver` is an AutoPlugin that is all that is required.
 | when there are no commits, or the project isn't a git repo           | HEAD+20140707-1030             | true       |
 ```
 
+## Tag Requirements
+
+In order to be recognized by sbt-dynver, tags must meet the following two requirements:
+1. Must be an annotated tag (as opposed to a lightweight tag)
+2. Must begin with the letter 'v'
```

Comment:
Let's be really specific :) It must begin with the letter 'v' and a digit: `v[0-9]`.